app/locales/en/messages.po

```diff
@@ -647,6 +647,10 @@ msgstr "Preview using medium width"
 msgid "controls.layout.preview-sm"
 msgstr "Preview using small width"
 
+#: app/configurator/components/preview-breakpoint.tsx
+msgid "controls.layout.preview-xl"
+msgstr "Preview using extra large width"
```

Comment:
I think it would be easier for the user to see 

Preview using mobile
Preview using tablet
Preview using small desktop screen
Preview using large desktop screen

Reply:
cc @KerstinFaye as these can be now edited via Accent (I used the ones from the design) – but agree that it would be more obvious 👍

Reply:
@ptbrowne We had this kind of naming before. I changed it when I thought about the embed mode which is not always full width, even when we look at it on a large screen. But maybe it's more intuitive with the old names. I can change it back.